code/game/machinery/suit_storage_unit.dm

```diff
@@ -1,184 +1,406 @@
-//////////////////////////////////////
 // SUIT STORAGE UNIT /////////////////
-//////////////////////////////////////
-
-
 /obj/machinery/suit_storage_unit
-	name = "Suit Storage Unit"
+	name = "suit storage unit"
 	desc = "An industrial U-Stor-It Storage unit designed to accomodate all kinds of space suits. Its on-board equipment also allows the user to decontaminate the contents through a UV-ray purging cycle. There's a warning label dangling from the control pad, reading \"STRICTLY NO BIOLOGICALS IN THE CONFINES OF THE UNIT\"."

```

Comment:
Maybe capitalise the `unit` in the name? IMO the name is also pretty horrible but that could just be me

Reply:
Why? When someone points to it for example it'll say

X points to the suit storage Unit.

Reply:
Oh, I didn't touch it, I assumed it was a /lore/ name `U-Stor-IT Storage`